File: Java/src/lab8/GameTest.java
package lab8;

public class GameTest {
	public static void main(String[] args){
		
    CrapsGame game1 = new CrapsGame();
    
    for (int counter = 1; counter <= 10; counter++)
    {
        game1.playGame(); 
    }


	}
}

File: C#/lab9/lab9/GameTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace lab9
{
    class GameTest
    {
        static void Main(string[] args)
        {
            CrapsGame game1 = new CrapsGame();
            for (int counter = 1; counter <= 10; counter++)
            {
                game1.PlayGame(); 
            }
        }
    }
}
